src/sagas/EntitySetsApiSagas.js

```diff
@@ -375,6 +377,45 @@ function* getPropertyTypeMetaDataForEntitySetsWatcher() :Saga<*> {
   yield takeEvery(GET_PROPERTY_TYPE_METADATA_FOR_ENTITY_SETS, getPropertyTypeMetaDataForEntitySetsWorker);
 }
 
+/*
+ *
+ * EntitySetsApi.updateEntitySetMetadata
+ * EntitySetsApiActions.updateEntitySetMetadata
+ *
+ */
+
+function* updateEntitySetMetadataWorker(action :SequenceAction) :Saga<WorkerResponse> {
+
+  if (!isValidAction(action, UPDATE_ENTITY_SET_METADATA)) {
+    return { error: new Error(ERR_INVALID_ACTION) };
+  }
+
+  let workerResponse :WorkerResponse;
+  const { id, value } = action;
+
+  try {
+    yield put(updateEntitySetMetadata.request(id, value));
+    const { entitySetId, update } = value;
```

Comment:
Please change `update` to `metadata` to be consistent with other APIs:

https://github.com/openlattice/lattice-sagas/blob/develop/src/sagas/EntityDataModelApiSagas.js#L1068
https://github.com/openlattice/lattice-sagas/blob/develop/src/sagas/EntityDataModelApiSagas.js#L1107